_posts/2021-04-08-alekos-bdk.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Alekos Filini on Bitcoin Development Kit"
+permalink: /blog/2021/04/08/bitcoin-development-kit/
+layout: post
+author: alekos
+name: "Alekos"
+alt: Alekos Filini on Bitcoin Development Kit
+category: "News"
+description: Alekos Filini on Bitcoin Development Kit.
+---
+
+<img src="/assets/images/2021-04-08-bitcoin-development-kit.svg" />
+
+_Alekos’ grant is generously supported by [Kraken][kraken website]._
+
+While a very high level understanding of Bitcoin can be enough to grasp how
+value is moved across the network, taking this knowledge and using it to build a
+real, user-ready product can prove to be very hard, due to the high security
+standards that need to be met, and also in part because of all the little quirks
+the Bitcoin protocol has.
+
+Over the years many different libraries have been released to help. Application
+developers can work more effectively on their products. Those libraries come in a
+wide variety of programming languages, but they have always historically
+suffered from either a lack of generalization, meaning that they would only
+support a very limited set of operations, or they would trade some of the
+convenience for a more powerful API, making them especially tricky to use in
+practice without a deep understanding of the protocol.
+
+This ultimately has reduced the “diversity” of Bitcoin
+products and wallets available to the public today: the vast majority of them
+only support single-signature schemes, or at most simple multi-signatures. More
+complex scripts that involve, for instance, timelocks are very rarely seen in
+the wild.
+
+The advent of output descriptors promises to remove once and for all the
+trade-off between usability and power: descriptors can be used to express with a
+very concise syntax even the most complex Bitcoin spending conditions. With the
+right tools, a developer can build a product that relies on arbitrary spending
+policies without having to ever touch raw Bitcoin scripts or transactions.
+
+Even better, software built with descriptors in mind can be very easy to adapt
+to new protocol changes, like the upcoming Taproot soft-fork, and it allows
+prototyping and shipping new features much more quickly because most of the
```

Comment:
```suggestion
to new protocol changes, like the upcoming Taproot soft-fork. Descriptors also allow
prototyping and shipping new features much more quickly because most of the
```